lib/surrealist.rb

```diff
@@ -148,7 +148,8 @@ def build_schema(instance:, **args)
     #   provided in the object's class. Values will be taken from the return values
     #   of appropriate methods from the object.
     def __build_schema(object, **args)
-      if (serializer = Surrealist::VarsHelper.find_serializer(object.class))
+      serializer = Surrealist::VarsHelper.find_serializer(object.class, tag: args[:tag])
+      if serializer
```

Comment:
I think you forgot to apply new feature to `#build_schema` method.

Reply:
There is no external serializer support for this method

Reply:
Yeah, I guess I missed it in #61.